Cisco/cisco-secure-firewall/tests/test_group_10.json

```diff
@@ -0,0 +1,39 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "Cisco Secure Firewall Threat Defense",
+        "dialect_uuid": "860ac698-0772-41a3-8007-cb74044fe443"
+      }
+    },
+    "message": "Task ran for 100 msec, Process = aaa_shim_thread, PC = abb111cc, Call stack =   0x000000aaab89d6a0  0x000000aaab88cdec  0x000000aaab88cd68"
+  },
+  "expected": {
+    "message": "Task ran for 100 msec, Process = aaa_shim_thread, PC = abb111cc, Call stack =   0x000000aaab89d6a0  0x000000aaab88cdec  0x000000aaab88cd68",
+    "event": {
+      "category": ["network"],
+      "kind": "event"
+    },
+    "action": {
+      "target": "network-traffic"
+    },
+    "cisco": {
+      "call_stack": "0x000000aaab89d6a0  0x000000aaab88cdec  0x000000aaab88cd68",
+      "ftd": {
+        "event": {
+          "duration": "100"
+        }
+      },
+      "message": "Task ran for 100 msec"
+    },
+    "host": {
+      "name": "abb111cc"
+    },
```

Comment:
According to [this documentation](https://www.cisco.com/c/en/us/td/docs/security/asa/syslog/b_syslog/syslogs7.html#ariaid-title27), `PC` is not the hostname but the `Instruction pointer of the CPU hogging process`. So, I think a custom field is more suitable for this value.